octopoes/octopoes/models/ooi/reports.py

```diff
@@ -39,10 +39,23 @@ class Report(OOI):
 
     observed_at: datetime
     parent_report: Reference | None = ReferenceField("Report", default=None)
+    report_recipe: Reference | None = ReferenceField("ReportRecipe", default=None)
     has_parent: bool
 
     _natural_key_attrs = ["report_id"]
 
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return f"Report {reference.tokenized.report_id}"
+
+
+class ReportRecipe(OOI):
+    recipe_id: UUID
+    object_type: Literal["ReportRecipe"] = "ReportRecipe"
+    cron: str
```

Comment:
I think `cron_expression` is a more appropriate name

Reply:
@underdarknl I think we need to remove this attribute from the recipe. It woudl be better if this lives in the scheduler. We can always pull all schedules and change the cron expressiom at the scheduler, otherwise f there is an update we need to update this variable in 2 places.

Reply:
as discussed, it stays in the ooi.